### DISCUSSION TITLE
Error: spawn /home/jack/go_workspace/lotus/lotus/lotus-miner ENOENT

### DISCUSSION THREAD

**Discussion by 815943555** [filecoin-project/lotus#3922] [General Q&A] [answered]
Error: spawn /home/jack/go_workspace/lotus/lotus/lotus-miner ENOENT
PM2      |     at _errnoException (util.js:1022:11)
PM2      |     at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
PM2      |     at onErrorNT (internal/child_process.js:372:16)
PM2      |     at _combinedTickCallback (internal/process/next_tick.js:138:11)
PM2      |     at process._tickDomainCallback (internal/process/next_tick.js:218:9)
PM2      | Change detected on path extern/filecoin-ffi/rust/target/release/build/openssl-sys-5526c27ccfcce9c7/out/openssl-build/install for app lotus miner - restarting
^C

**Comment by jennijuju** [comment]
Not enough information provided as an issue, moving to discussion for further discussion & trouble shooting

**Comment by SgtCoin** [comment]
@815943555 Is this still an issue?

**Comment by SgtCoin** [comment, marked as answer]
Archiving for future reference due to age.
